backend/app/resources.py

```diff
@@ -0,0 +1,109 @@
+from flask_restful import Resource, marshal_with, marshal
+from sqlalchemy import exc
+
+from app.marshallers import item_marshaller, category_marshaller, subcategory_marshaller
+from app.models import Item, Category, Subcategory
+from app.parsers import item_parser, subcategory_parser, category_parser, creating_item_parser, \
+    creating_category_parser, creating_subcategory_parser
+from init_app import db
+
+
+def add_to_database(item):
+    try:
+        db.session.add(item)
+        db.session.commit()
+        return True
+    except exc.IntegrityError:
+        db.session.rollback()
+        return False
+
+
+class Items(Resource):
+    @marshal_with(item_marshaller)
+    def get(self, item_id):
+        return Item.query.get_or_404(item_id)
+
+    @marshal_with(item_marshaller)
+    def delete(self, item_id):
+        item = Item.query.get_or_404(item_id)
+        db.session.delete(item)
+        db.session.commit()
+        return '', 204
+
+    @marshal_with(item_marshaller)
+    def put(self, item_id):
+        item = Item.query.get_or_404(item_id)
```

Comment:
you can have method which could be get object etc, inspired by Django's ORM

Reply:
like get(object), where should I use it?